android/app/src/main/java/com/zulipmobile/notifications/FCMPushNotifications.kt

```diff
@@ -179,10 +213,18 @@ private fun createSummaryNotification(
     }
 }
 
+/** The unique tag we use for the group of notifications addressed to this Zulip account. */
 private fun extractGroupKey(identity: Identity): String {
     return "${identity.realmUri.toString()}|${identity.userId}"
```

Comment:
Good question. It can't, because `|` is a character that can't appear in a URL. That was discussed in the PR thread that led to this, but didn't make it into a comment; it should.